### PR TITLE
remove v1 from ui module

### PR DESCRIPTION
**Pull request by abhishek1508**
## Description

Remove `v1` from ui module. (Fixes [#502](https://github.com/mapbox/navigation-sdks/issues/502))

- [x] I have added any issue links
- [x] I have added all related labels (`bug`, `feature`, `new API(s)`, `SEMVER`, etc.)
- [x] I have added the appropriate milestone and project boards

### Goal

Remove `v1` from ui module.

### Implementation

Remove `v1` from ui module.

## Screenshots or Gifs

Please include all the media files to give some context about what's being implemented or fixed. It's not mandatory to upload screenshots or gifs, but for most of the cases it becomes really handy to get into the scope of the feature / bug being fixed and also it's REALLY useful for UI related PRs screenshot gif

## Testing

Please describe the manual tests that you ran to verify your changes

- [x] I have tested locally (including `SNAPSHOT` upstream dependencies if needed) through testapp/demo app and run all activities to avoid regressions
- [x] I have tested via a test drive, or a simulation/mock location app
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have updated the `CHANGELOG` including this PR
- [x] We might need to update / push `api/current.txt` files after running `$> make core-update-api` (Core) / `$> make ui-update-api` (UI) if there are changes / errors we're 🆗 with (e.g. `AddedMethod` changes are marked as errors but don't break SemVer) 🚀 If there are SemVer breaking changes add the `SEMVER` label. See [Metalava](https://github.com/mapbox/mapbox-navigation-android/blob/master/docs/metalava.md) docs

### CODE REVIEW

File: libnavigation-ui/gradle.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
 RELEASE_SDK = ui
-POM_ARTIFACT_ID=ui-v1
+POM_ARTIFACT_ID=v1
```
Comment (LukasPaczos) [inline libnavigation-ui/gradle.properties:2]:
```suggestion
POM_ARTIFACT_ID=ui
```
:)

Comment (abhishek1508) [reply libnavigation-ui/gradle.properties:2]: haha. 😂

**Comment by Guardiola31337** [comment]
When working on the latest release noticed that we may consolidate the CI jobs into one. That'll remove the need of creating two tags and releasing the artifacts separately simplifying a bit the overall release process https://github.com/mapbox/mapbox-navigation-android/blob/master/docs/publishing.md#artifact-distribution

What do you think @LukasPaczos @abhishek1508?

**Comment by LukasPaczos** [comment]
Sounds good to me :+1:

**Pull request merged**